src/interfaces/incorporation-interfaces/incorporationFiling-interface.ts

```diff
@@ -0,0 +1,21 @@
+import { IncorporationApplicationIF } from '@/interfaces/incorporation-interfaces'
+
+/** Incorporation Application filing loaded from / saved to the Legal API. */
+export interface IncorporationFilingIF {
```

Comment:
Maybe don't need this.

Reply:
I'd prefer to leave the interfaces until a time when we are importing them, if possible :)

Reply:
What do the other reviewers think?

Reply:
All good, i don't disagree! Will remove :)